docs/r/cdap_namespace.md

```diff
@@ -0,0 +1,10 @@
+# cdap_namespace
+
+## Argument Reference
+
+The following fields are supported:
+
+* name:
```

Comment:
what about 

name (Required):
The name of the namespace.

Reply:
Done.